include/class.config.php

```diff
@@ -673,19 +673,20 @@ function alertAcctManagerONNewMessage() {
         return ($this->get('message_alert_acct_manager'));
     }
 
-    function alertONNewNote() {
+    //TODO: change note_alert to activity_alert
```

Comment:
I think we should change `note_alert_*` to `activity_alert_*` to ward-off confusions going forward.